lib/lightning/version_control/github_client.ex

```diff
@@ -126,11 +67,22 @@ defmodule Lightning.VersionControl.GithubClient do
           ]}
        ])}
     else
-      %{status: 404} = err ->
-        {:error, :installation_not_found, err}
-
-      _unused_status ->
-        {:error, :invalid_pem}
+      %{status: 404, body: body} = resp ->
+        error =
+          GithubError.installation_not_found(
+            "Github Installation APP ID is misconfigured",
+            body
+          )
+
+        Sentry.capture_exception(error)
+        Logger.error(inspect(resp, label: "Unexpected Github Response: "))
+        {:error, error}
+
+      resp ->
+        Logger.error(inspect(resp, label: "Unexpected Github Response: "))
+        error = GithubError.invalid_pem("Github Cert is misconfigured")
```

Comment:
this is not necessarily an invalid pem. you should instead send "Error" with no misleading description.